install-dependencies.sh

```diff
@@ -69,15 +69,10 @@ fedora_packages=(
     maven
     patchelf
     python3
-    python3-aiohttp
     python3-pip
     python3-magic
     python3-colorama
     python3-tabulate
-    python3-boto3
-    python3-pytest
-    python3-pytest-asyncio
-    python3-redis
```

Comment:
Why do this? If they work, leave them alone. Installing from pip pollutes developer's namespace (remember that install-dependencies.sh is also used by developers, not everyone uses dbuild).

Reply:
We need specifically to get python3-pytest-asyncio to be obtained from pip.
If anything, this list could be moved to requirements.txt.

Reply:
Then change only this package. If you really must (I believe you don't, but I don't have time to fight the version-dependency fight while on vacation). Why change all the packages?